src/PKSim.Infrastructure/Services/MoBiExportTask.cs

```diff
@@ -148,6 +148,17 @@ private void exportSimulationToFile(Simulation simulation, string moBiFile)
       private void createSimulationProjectSnapshot(Simulation simulation, IModelCoreSimulation modelCoreSimulation)
       {
          modelCoreSimulation.Configuration.ModuleConfigurations.First().Module.Snapshot = _simulationToProjectSnapshotMapper.MapFrom(simulation);
+
+         if (simulation.Individual != null && modelCoreSimulation.Configuration.Individual != null)
```

Comment:
feels like we have no idea what's going on with the code here. We have a PKSim simulation. 
If the Individual is null, themodelCoreSimulation.Configuration.Individual will be null and vice versa. 

doens't _simulationToProjectSnapshotMapper.MapFrom(simulation); somehow has the indiviudal snapshot also ? We are doing it twice it seems

Reply:
yes, if I refactor this a bit they can all be done in one go. Less of a mapper, more of a task that adds the snapshots to the ModelCoreSimulation where needed